org.scala-ide.sdt.core/src/org/scalaide/core/internal/jdt/model/ScalaSourceFile.scala

```diff
@@ -33,58 +36,60 @@ class ScalaSourceFileProvider extends SourceFileProvider {
 
 object ScalaSourceFile {
 
-  /** Considering [[org.eclipse.jdt.internal.core.util.HandleFactory]] isn't thread-safe, and because
-   *  `ScalaSourceFile#createFromPath` can be called concurrently from different threads, using a
-   *  `ThreadLocal` ensures that a `HandleFactory` instance is never shared across threads.
+  /**
+   * Considering [[org.eclipse.jdt.internal.core.util.HandleFactory]] isn't thread-safe, and because
+   * `ScalaSourceFile#createFromPath` can be called concurrently from different threads, using a
+   * `ThreadLocal` ensures that a `HandleFactory` instance is never shared across threads.
    */
   private val handleFactory: ThreadLocal[HandleFactory] = new ThreadLocal[HandleFactory] {
     override protected def initialValue(): HandleFactory = new HandleFactory
   }
 
-  /** Creates a Scala source file handle if the given resource path points to a scala source.
-   *  The resource path is a path to a Scala source file in the workbench (e.g. /Proj/a/b/c/Foo.scala).
+  /**
+   * Creates a Scala source file handle if the given resource path points to a scala source.
+   * The resource path is a path to a Scala source file in the workbench (e.g. /Proj/a/b/c/Foo.scala).
    *
-   *  @note This assumes that the resource path is the toString() of an `IPath`.
+   * @note This assumes that the resource path is the toString() of an `IPath`.
    *
-   *  @param path Is a path to a Scala source file in the workbench.
+   * @param path Is a path to a Scala source file in the workbench.
    */
-  def createFromPath(path: String) : Option[ScalaSourceFile] = {
+  def createFromPath(path: String): Option[ScalaSourceFile] = {
     if (!path.endsWith(".scala"))
       None
     else {
       // Always `null` because `handleFactory.createOpenable` is only called to open source files, and the `scope` is not needed for this.
       val unusedScope = null
       val source = handleFactory.get().createOpenable(path, unusedScope)
       source match {
-        case ssf : ScalaSourceFile => Some(ssf)
-        case _ => None
+        case ssf: ScalaSourceFile ⇒ Some(ssf)
+        case _                    ⇒ None
```

Comment:
this formatting done by tool or you? generally I don't think this kind of formatting is cool